packages/commonwealth/client/scripts/views/pages/CommunityGroupsAndMembers/Members/MembersSection/MembersSection.tsx

```diff
@@ -77,13 +86,21 @@ const MembersSection = ({
           },
           groups: {
             sortValue: member.groups
+              .map((group) => group.name)
               .sort((a, b) => a.localeCompare(b))
               .join(' ')
               .toLowerCase(),
             customElement: (
               <div className="table-cell">
                 {member.groups.map((group, index) => (
-                  <CWTag key={index} label={group} type="referendum" />
+                  <div key={index} className="group-item">
+                    <span className="group-name">{group.name}</span>
+                    <img
+                      src={group.groupImageUrl}
+                      alt={group.name}
+                      className="group-image"
+                    />
```

Comment:
lets also add a fallback img (using one of the default images) in case if there is no group image.